doc/reference/network_ovn.md

```diff
@@ -5,44 +5,78 @@ discourse: 11033
 (network-ovn)=
 # OVN network
 
-The ovn network type allows the creation of logical networks using the OVN SDN. This can be useful for labs and
-multi-tenant environments where the same logical subnets are used in multiple discrete networks.
+<!-- Include start OVN intro -->
+{abbr}`OVN (Open Virtual Network)` is a software-defined networking system that supports virtual network abstraction.
+You can use it to build your own private cloud.
+See https://www.ovn.org/ for more information.
+<!-- Include end OVN intro -->
 
-A LXD OVN network can be connected to an existing managed LXD bridge network in order for it to gain outbound
-access to the wider network. All connections from the OVN logical networks are NATed to a dynamic IP allocated by
-the parent network.
+The `ovn` network type allows to create logical networks using the OVN {abbr}`SDN (software-defined networking)`.
+This kind of network can be useful for labs and multi-tenant environments where the same logical subnets are used in multiple discrete networks.
 
-```{toctree}
-:maxdepth: 1
+A LXD OVN network can be connected to an existing managed {ref}`network-bridge` or {ref}`network-physical` to gain outbound access to the wider network.
+All connections from the OVN logical networks are NATed to a dynamic IP allocated by the parent network.
 
-Set up OVN </howto/network_ovn_setup>
-Create routing relationships </howto/network_ovn_peers>
-```
+See {ref}`network-ovn-setup` for basic instructions for setting up an OVN network.
 
 (network-ovn-options)=
 ## Configuration options
 
+The following configuration key namespaces are currently supported for the `ovn` network type:
+
+ - `bridge` (L2 interface configuration)
+ - `dns` (DNS server and resolution configuration)
+ - `ipv4` (L3 IPv4 configuration)
+ - `ipv6` (L3 IPv6 configuration)
+ - `security` (network ACL configuration)
```

Comment:
Is it correct that OVN doesn't support maas?
Before, the docs said that all network types support it.

Reply:
Yes it cannot register the subnet in MAAS as it doesn't use a MAAS subnet, but rather its own private overlay subnet.